app/lib/constants/colors.ts

```diff
@@ -254,6 +268,13 @@ export const colors = {
 		gray100: '#CBCED1',
 		n900: '#FFFFFF',
 		overlayColor: '#1F2329B2',
+		audioPlayerPrimary: '#3976D1',
+		audioPlayerSecondary: '#404754',
+		audioRateBackground: '#353B45',
+		audioRateText: '#E4E7EA',
+		audioTimeText: '#E4E7EA',
+		audioBorderColor: '#2F343D',
```

Comment:
In these cases use the name that is there in figma, this color for example its name is: Theme/Stroke/extra-light